examples/hybrid/sphere/hadley_circulation.jl

```diff
@@ -0,0 +1,296 @@
+    else
+        q1 = 0.0
+    end
+
+    ρq1 = ρ_ref(z) * q1
+
+    return (ρ = ρ_ref(z), ρq1 = ρq1)
+end
+
+# IC FieldVector
+y0 = Fields.FieldVector(ρ = y0.ρ, ρq1 = y0.ρq1)
+
+ystar = copy(y0)
+parameters = (;
+    coords = coords,
+    face_coords = face_coords,
+    τ = τ,
+    u₀ = u₀,
+    ω₀ = ω₀,
+    K = K,
+    ρ₀ = ρ₀,
+)
+
+# Set up the initial flow
+rhs!(ystar, y0, parameters, 0.0, dt, 1)
+
+# run!
+prob = ODEProblem(IncrementingODEFunction(rhs!), copy(y0), (0.0, T), parameters)
+sol = solve(
+    prob,
+    SSPRK33ShuOsher(),
+    dt = dt,
+    saveat = dt,
+    progress = true,
+    adaptive = false,
+    progress_message = (dt, u, p, t) -> t,
+)
+
+q1_error =
+    norm(sol.u[end].ρq1 ./ ρ_ref.(coords.z) .- y0.ρq1 ./ ρ_ref.(coords.z)) /
+    norm(y0.ρq1 ./ ρ_ref.(coords.z))
+@test q1_error ≈ 0.0 atol = 0.05
+
+Plots.png(
+    Plots.plot(
+        sol.u[trunc(Int, end / 2)].ρq1 ./ ρ_ref.(coords.z),
+        level = 11,
+        clim = (-0.1, 3.5),
+    ),
+    joinpath(path, "q1_half_day_level_11.png"),
+)
+
+anim = Plots.@animate for u in sol.u
+    Plots.plot(u.ρq1 ./ ρ_ref.(coords.z), level = 11, clim = (-0.1, 3.5))
+end
+Plots.mp4(anim, joinpath(path, "q1_anim_level_11.mp4"), fps = 10)
+
+# Remapping infrastructure to have lat-altitude Plots
+Nq = Spaces.Quadratures.degrees_of_freedom(
+    Spaces.quadrature_style(hv_center_space),
+)
+datafile_cc = "hadley_circulation.nc"
+NCDataset(datafile_cc, "c") do nc
+    # defines the appropriate dimensions and variables for a space coordinate
+    def_space_coord(nc, hv_center_space, type = "cgll")
+    def_space_coord(nc, hv_face_space, type = "cgll")
+    # defines the appropriate dimensions and variables for a time coordinate (by default, unlimited size)
+    nc_time = def_time_coord(nc)
+
+    # define variables
+    nc_rho = defVar(nc, "rho", Float64, hv_center_space, ("time",))
+    nc_q1 = defVar(nc, "q1", Float64, hv_center_space, ("time",))
+
+    # write data to netcdf file
+    for i in 1:length(sol.u)
+        nc_time[i] = sol.t[i]
+
+        # write fields to file
+        nc_rho[:, i] = sol.u[i].ρ
+        nc_q1[:, i] = sol.u[i].ρq1 ./ sol.u[i].ρ
+    end
+end
+
+# write out our cubed sphere mesh
+meshfile_cc = "mesh_cubedsphere.g"
+write_exodus(meshfile_cc, hv_center_space.horizontal_space.topology)
+
+# write out RLL mesh
+nlat = 90
+nlon = 180
+meshfile_rll = "mesh_rll.g"
+rll_mesh(meshfile_rll; nlat = nlat, nlon = nlon)
+
+# construct overlap mesh
+meshfile_overlap = "mesh_overlap.g"
+overlap_mesh(meshfile_overlap, meshfile_cc, meshfile_rll)
+
+# construct remap weight file
+weightfile = "remap_weights.nc"
+remap_weights(
+    weightfile,
+    meshfile_cc,
+    meshfile_rll,
+    meshfile_overlap;
+    in_type = "cgll",
+    in_np = Spaces.Quadratures.degrees_of_freedom(
+        Spaces.quadrature_style(hv_center_space),
+    ),
+)
+
+# apply remap
+datafile_rll = "data_rll.nc"
+apply_remap(datafile_rll, datafile_cc, weightfile, ["rho", "q1"])
```

Comment:
Here is where the remap be performed and you want to analyze data_rll instead

Reply:
Thank you! I fixed it